README.md

```diff
@@ -1 +1,70 @@
-# microV-wk1proj
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project 1 Positioning and FLoating Elements
+
+> Week 1 project of Microverse.
+
+<img src="img/ScreeShot-wk1Proj.png" width="700">
+
+This project is a clone of a New York Times article.
+
+## Built With
+
+- HTML
+- CSS
+
+## Live Demo
+
+[Live Demo Link](https://memelopez.github.io/microV-wk1proj/)
+
+## Getting Started
+
+**it's easy.**
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+### Setup
+
+### Install
+
+### Usage
+
+### Run tests
+
+### Deployment
+
```

Comment:
- [ ] [REQUIRED] Please provide the steps to take in order for the project to be setup. If no steps are required, you should remove these headings.